src/StructuredLogger.Tests/StructuredLogger.Tests.csproj

```diff
@@ -32,4 +33,9 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
+  <PropertyGroup Condition=" '$(OS)' == 'Windows_NT' ">
+    <SignAssembly>True</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\StructuredLogger\key.snk</AssemblyOriginatorKeyFile>
+    <DelaySign>False</DelaySign>

```

Comment:
I'm guessing this is for IVT. Which API is internal that you'd like to use from tests?

Reply:
Correct.
`BuildEventArgsReader` is needed (to verify the equality of full roundtrip through reader and writer - as that's needed for redacting).

Reply:
Oh, and we can't make that public because otherwise it'll conflict with the one in the actual MSBuild for third-parties? Makes sense.

Reply:
Correct.
We can possibly place it in different namespace to avoid the conflict. But not exposing at all feels as a better way.